MAUI/DataForm/layout.md

```diff
@@ -398,4 +398,27 @@ private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs
 {% endhighlight %}
 {% endtabs %}
 
-Here, the `Name` field will be hidden.
\ No newline at end of file
+Here, the `Name` field will be hidden.
+
+## Change the editor height
+
+The data form editor height is changed by using the `EditorHeight` property of the [DataFormItem](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.DataFormItem.html).
```

Comment:
The .NET MAUI DataForm allows you to modify the height of each editor to fit the height of the data form or to customize the height based on text size by using the `EditorHeight` property of

Reply:
content changed